flink-cdc-connect/flink-cdc-source-connectors/flink-connector-oracle-cdc/src/main/java/org/apache/flink/cdc/connectors/oracle/table/OracleTableSourceFactory.java

```diff
@@ -195,8 +201,12 @@ public Set<ConfigOption<?>> optionalOptions() {
     private static final String SCAN_STARTUP_MODE_VALUE_INITIAL = "initial";
     private static final String SCAN_STARTUP_MODE_VALUE_SNAPSHOT = "snapshot";
     private static final String SCAN_STARTUP_MODE_VALUE_LATEST = "latest-offset";
+    private static final String SCAN_STARTUP_MODE_VALUE_SPECIFIC_OFFSETS = "specific-offset";
+    private static final String SCAN_STARTUP_MODE_VALUE_SPECIFIC_OFFSETS_PREFIX =
+            "scan.startup.specific-offset.";
```

Comment:
Do we have other kind of offset other than `lsn` in Oracle?
If `lsn` is the only kind of offset, we can upgrade this into a ConfigOption.

Reply:
Please update related doc for the newly added startup option.